src/pages/Training/AddEventModal/AddEventModal.tsx

```diff
@@ -37,23 +38,44 @@ type UserType = {
   email: string;
 };
 
+const schema = yup.object({
+  title: yup.string().required('Tytuł jest wymagany').min(3, 'Tytuł musi mieć co najmniej 3 znaki'),
+  description: yup.string().required('Opis jest wymagany').min(3, 'Opis musi mieć co najmniej 3 znaki'),
+  guests: yup.array().default([]),
+  time: yup.string().required('Godzina rozpoczęcia jest wymagana'),
+  endTime: yup
+    .string()
+    .required('Godzina zakończenia jest wymagana')
+    .test('is-later', 'Ten sam czas zakończenia!', function (endTime) {
+      const startTime = this.parent.time;
+      return startTime && endTime ? startTime < endTime : true;
+    }),
```

Comment:
```suggestion
    .test('is-later', 'Ten sam czas zakończenia!', (endTime) => {
      const startTime = this.parent.time;
      return startTime && endTime ? startTime < endTime : true;
    }),
```
nie przypominam sobie, zebysmy uzywali funkcji pre ES6